script/LocalDeploy.s.sol

```diff
@@ -92,6 +95,14 @@ contract LocalDeploy is SymbioticCoreInit {
     uint208 internal immutable NUM_AGGREGATORS = uint208(vm.envOr("NUM_AGGREGATORS", uint256(1)));
     uint208 internal immutable NUM_COMMITTERS = uint208(vm.envOr("NUM_COMMITTERS", uint256(1)));
 
+    // CREATE3 salts
```

Comment:
the goal of it was to have deterministic addresses for local deployment, so can we have these addresses?